bedrock/contentful/api.py

```diff
@@ -613,7 +766,7 @@ def get_content(self):
             entry_obj = self.page.fields()["entry"]
         elif entry_type.startswith("page"):  # WARNING: this requires a consistent naming of page types in Contentful, too
             entry_obj = self.page
-            seo_obj = getattr(self.page, "seo", None)
+            seo_obj = getattr(self.page, "seo", None)  # WARNING: this also requires a consistent ID-naming of SEO field in Contentful, too
```

Comment:
That comment reads like something I think I suggested via Slack. Trying a clearer version:

```suggestion
            seo_obj = getattr(self.page, "seo", None)  # WARNING: Ensure any page in Contentful linking to an SEO Metadata object names the field `seo`
```